dist/common/scripts/scylla_coredump_setup

```diff
@@ -73,11 +73,11 @@ Options=bind
 [Install]
 WantedBy=multi-user.target
 '''[1:-1]
-            with open('/etc/systemd/system/var-lib-systemd-coredump.mount', 'w') as f:
+            with open('/etc/systemd/system/var-lib-scylla-coredump.mount', 'w') as f:
                 f.write(dot_mount)
             makedirs('/var/lib/scylla/coredump')
             systemd_unit.reload()
-            systemd_unit('var-lib-systemd-coredump.mount').enable()
+            systemd_unit('var-lib-scylla-coredump.mount').enable()
         if os.path.exists('/usr/lib/sysctl.d/50-coredump.conf'):
             run('sysctl -p /usr/lib/sysctl.d/50-coredump.conf')
```

Comment:
I'm confused, shouldn't we bind-mount /var/lib/scylla/coredump into /var/lib/systemd/coredump? So we get the filesystem with lots of space mounted into /var/lib/systemd/coredump?

Should we switch What and Where instead?

Reply:
I am also confused, we just switched them around in e77dad3adf7051a489223bbd55ecbc461b377421. @syuu1228 @bentsi

Reply:
https://github.com/scylladb/scylla/commit/e77dad3adf7051a489223bbd55ecbc461b377421 is a wrong fix.
I reverted it by https://github.com/scylladb/scylla/commit/9d9d54c804dd400d90edfdb84c8f5319d06b1633